.github/FUNDING.yml

```diff
@@ -0,0 +1 @@
+github: ai
```

Comment:
Nope. This is a funding page for my personal projects (it adds Sponsor button).

We don’t need it here.